contrib/systemd/system/podman-image-prune.service.in

```diff
@@ -0,0 +1,10 @@
+[Unit]
+Description=Podman image prune service
+Documentation=man:podman-image-prune(1)
+
+[Service]
+Type=oneshot
+ExecStart=@@PODMAN@@ image prune -a -f
+
+[Install]
+WantedBy=default.target
```

Comment:
This should not have a install section, only the timer should be enabled

Reply:
Thanks for the review. I just copied the existing `podman-auto-update.service.in` service which also has an Install section.
Removed in ed681454a0